<launch>
	<arg name="port" default="/dev/ttyUSB0"/>
	<arg name="cube_id" default="1"/>

	<node pkg="turn_table_interface" type="turn_table_interface" name="turn_table_interface" output="screen">

		<!-- REMEMBER TO CHECK THE PORT * -> /dev/ttyUSB*!/-->
		<!-- AND THEN REMEMBER TO sudo chmod 777 /dev/ttyUSB*!/-->
		<param name="port" value="$(arg port)"/>

		<!-- 0 for broadcasting, cube_id otherwise/-->
		<param name="cube_id" value="$(arg cube_id)"/>
	</node>
</launch>

